cluster/service/cluster_service.go

```diff
@@ -328,3 +328,20 @@ func (c clusterService) createIdentityCluster(ctx context.Context, identityID, c
 		return nil
 	})
 }
+
+// RemoveIdentityToClusterLink removes Identity to Cluster link/relation
+func (c clusterService) RemoveIdentityToClusterLink(ctx context.Context, identityID uuid.UUID, clusterURL string) error {
+	rc, err := c.Repositories().Clusters().LoadClusterByURL(ctx, clusterURL)
+	if err != nil {
+		log.Error(ctx, map[string]interface{}{
+			"cluster_url": clusterURL,
+			"err":         err,
+		}, "failed to load cluster with url %s", clusterURL)
+		return errors.NewBadParameterError("cluster-url", fmt.Sprintf("cluster with requested url %s doesn't exist", clusterURL))
+	}
+
+	clusterID := rc.ClusterID
```

Comment:
no need to assign a new var here, I believe.

Reply:
addressed in 83f9362